docker.Makefile

```diff
@@ -63,7 +63,7 @@ binary: build_binary_native_image ## build the CLI
 build: binary ## alias for binary
 
 plugins: build_binary_native_image ## build the CLI plugin examples
-	docker run --rm $(ENVVARS) $(MOUNTS) $(BINARY_NATIVE_IMAGE_NAME) ./scripts/build/plugins
+	docker run --rm $(ENVVARS) $(DOCKER_CLI_MOUNTS) $(BINARY_NATIVE_IMAGE_NAME) ./scripts/build/plugins
```

Comment:
could we do the reverse, and remove the `DOCKER_CLI_` prefix from all of these? This is the makefile for the CLI repository, so not sure why we need the prefix

Reply:
I also noticed the `$DOCKER_CLI_SHELL` variable was added; alpine images now have `/bin/sh` set by default (previously they didn't, which was perhaps the reason this argument was added), but we install `bash` as well @wk8 was that variable added to switch to `bash`? If so, perhaps we should update the Dockerfile, and make it the default?

Reply:
@thaJeztah the `DOCKER_CLI` prefix is for the variables which can be overriden with env variables. Allows setting them in eg a shell profile (and that's also why they are nicely set apart - https://github.com/wk8/cli/blob/2178fea84dbea0b278d412b5f9cf16e3542d2650/docker.Makefile#L7-L11)

Reply:
shouldn't those be set _after_ `make` though? 

```
make FOO=bar BAR=baz <target>
```

Reply:
That's one way to do it but
```
export FOO=bar
make <target>
```

or
```
FOO=bar make <target>
```

work just the same.

Reply:
As to `$DOCKER_CLI_SHELL`, yup I've set mine to `bash --login` so that it also loads my shell profile - just made it `ash` in https://github.com/docker/cli/pull/1511 to make the change seamless, but I don't any reason why we couldn't switch to bash.

Reply:
I'd be +1 to change it to bash by default, but we could do that separate